Azure/sonic-buildimage
# sFlow: Upgrade hsflowd to 2.0.26-1

**- What I did**

Upgrade hsflowd to 2.0.26-1 : 
https://github.com/sflow/host-sflow/commit/53b02dadc3ef6c44a7aca87102138048a3a1ec62
https://github.com/sflow/host-sflow/commit/94dde90e9e290ef895677cbb621b47ad6d48ae8e

**- How I did it**

Update makefiles to get 2.0.26-1

**- How to verify it**

Restart unit (in a loop) with traffic when sflow is enabled and check hsflowd is stable and sampling is successful.

**- Description for the changelog**


Sometimes hsflowd crashes due to a race when sampling traffic during hsflowd startup.

**- A picture of a cute animal (not mandatory but encouraged)**
